### PR TITLE
Use "FCL" not "JavaScript SDK" in documentation.

### PR DESCRIPTION
**Pull request by 10thfloor**
Removes all mention of "JavaScript SDK" in favour of "FCL (Flow Client Library)"

### CODE REVIEW

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -22,12 +22,12 @@ const navConfig = {
       "Learn the basics of Flow and Cadence using our browser-based playground.",
     linkText: "Launch the Flow Playground",
   },
-  "JavaScript SDK": {
-    url: "/flow-js-sdk",
+  "FCL (Flow Client Library)": {
+    url: "/fcl",
```
Comment (10thfloor) [inline docs/gatsby-config.js:26]:
```suggestion
    url: "/fcl-js",
```

File: docs/content/faq/operators.mdx [hunk 2]
```diff
@@ -134,18 +134,18 @@ And there is an example of usage here:
...
-[https://github.com/onflow/flow-js-sdk/tree/master/packages/sdk#getevents-usage](https://github.com/onflow/flow-js-sdk/tree/master/packages/sdk#getevents-usage)
+[https://github.com/onflow/fcl-js/tree/master/packages/sdk#getevents-usage](https://github.com/onflow/fcl-js/tree/master/packages/sdk#getevents-usage)
 And there is an example of usage here:
 
 [https://github.com/onflow/kitty-items/tree/master/kitty-items-js/src/workers](https://github.com/onflow/kitty-items/tree/master/kitty-items-js/src/workers)
 
 ### Using Other Programming Languages
 
-Currently, we only provide Go and JavaScript SDKs for Flow. For an example of accessing the Flow blockchain and consuming events from it using another programming language, see "Interact with Flow using Ruby" by Daniel Podaru:
+Currently, we only provide Go and FCL (Flow Client Library)s for Flow. For an example of accessing the Flow blockchain and consuming events from it using another programming language, see "Interact with Flow using Ruby" by Daniel Podaru:
```
Comment (srinjoyc) [inline docs/content/faq/operators.mdx:148]:
```suggestion
Currently, we only provide Go and FCL-JS (Flow Client Library) for Flow. For an example of accessing the Flow blockchain and consuming events from it using another programming language, see "Interact with Flow using Ruby" by Daniel Podaru:
```

File: docs/content/faq/developers.mdx [hunk 3]
```diff
@@ -158,7 +158,7 @@ With FCL, you can:
 
 You can find details about accessing Flow mainnet here [https://docs.onflow.org/access-api](https://docs.onflow.org/access-api).
 
-The access is via a GRPC interface. Transactions can also be submitted using the GRPC [SendTransaction call](https://docs.onflow.org/access-api#sendtransaction). A GRPC client written in any language should be able to talk via the Access API. We have a [Go SDK](https://github.com/onflow/flow-go-sdk) and a [JavaScript SDK](https://github.com/onflow/flow-js-sdk) that can be used if you are using a Go based or a JS based client respectively.
+The access is via a GRPC interface. Transactions can also be submitted using the GRPC [SendTransaction call](https://docs.onflow.org/access-api#sendtransaction). A GRPC client written in any language should be able to talk via the Access API. We have a [Go SDK](https://github.com/onflow/flow-go-sdk) and a [FCL (Flow Client Library)](https://github.com/onflow/fcl-js) that can be used if you are using a Go based or a JS based client respectively.
```
Comment (srinjoyc) [inline docs/content/faq/developers.mdx:161]:
```suggestion
The access is via a GRPC interface. Transactions can also be submitted using the GRPC [SendTransaction call](https://docs.onflow.org/access-api#sendtransaction). A GRPC client written in any language should be able to talk via the Access API. We have a [Go SDK](https://github.com/onflow/flow-go-sdk) and [FCL-JS (Flow Client Library)](https://github.com/onflow/fcl-js) that can be used if you are using a Go based or a JS based client respectively.
```

File: docs/content/faq/developers.mdx [hunk 4]
```diff
@@ -130,15 +130,15 @@ And there is an example of usage here:
...
 
 This is documented here:
 
-[https://github.com/onflow/flow-js-sdk/tree/master/packages/sdk#getevents-usage](https://github.com/onflow/flow-js-sdk/tree/master/packages/sdk#getevents-usage)
+[https://github.com/onflow/fcl-js/tree/master/packages/sdk#getevents-usage](https://github.com/onflow/fcl-js/tree/master/packages/sdk#getevents-usage)
 
 ### Using Other Programming Languages
 
-Currently, we only provide Go and JavaScript SDKs for Flow. For an example of accessing the Flow blockchain and consuming events from it using another programming language, see "Interact with Flow using Ruby" by Daniel Podaru:
+Currently, we only provide Go and FCL (Flow Client Library)s for Flow. For an example of accessing the Flow blockchain and consuming events from it using another programming language, see "Interact with Flow using Ruby" by Daniel Podaru:
```
Comment (srinjoyc) [inline docs/content/faq/developers.mdx:141]:
```suggestion
Currently, we only provide Go and FCL-JS (Flow Client Library) for Flow. For an example of accessing the Flow blockchain and consuming events from it using another programming language, see "Interact with Flow using Ruby" by Daniel Podaru:
```

File: docs/compatibility.md [hunk 5]
```diff
@@ -98,7 +98,7 @@ Each Flow component exposes a public API, loosely defined as the functionality t
             </td>
         </tr>
         <tr>
-            <td>Flow JavaScript SDK</td>
+            <td>Flow FCL (Flow Client Library)</td>
```
Comment (srinjoyc) [inline docs/compatibility.md:101]:
```suggestion
            <td>FCL-JS (Flow Client Library)</td>
```

**Review by srinjoyc — APPROVED** [review]

File: docs/content/index.mdx [hunk 6]
```diff
@@ -4,19 +4,19 @@ description: "Welcome to the Flow Developer Portal"
...
 
 **Ready to start building?** Take a tutorial or have a look at an example application:
 
 ### 🏃‍♂️ App Quickstart Using React
 
-Follow the <TrackingLink href="/fcl/flow-app-quickstart/" eventName="Homepage_link_/fcl/flow-app-quickstart/_clicked">Flow App Quickstart</TrackingLink> for a detailed guide covering how to use the <TrackingLink href="/fcl/" eventName="Homepage_link_/fcl/_clicked">Flow JS SDK</TrackingLink> to build a serverless app that connects to Flow, using React!
+Follow the <TrackingLink href="/fcl/flow-app-quickstart/" eventName="Homepage_link_/fcl/flow-app-quickstart/_clicked">Flow App Quickstart</TrackingLink> for a detailed guide covering how to use the <TrackingLink href="/fcl/" eventName="Homepage_link_/fcl/_clicked">FCL-JS (Flow Client Library)</TrackingLink> to build a serverless app that connects to Flow, using React!
```
Comment (srinjoyc) [inline docs/content/index.mdx:19]: Some issues here.. link doesn't seem to be working on preview.

Comment (srinjoyc) [reply docs/content/index.mdx:19]: Some links are /fcl and some a /fcl-js

**Pull request merged**